src/Resources/Locales/en.json

```diff
@@ -654,6 +654,7 @@
                     "labelPlaceholder": "Enter label to show in chart",
                     "add": "Add time series",
                     "edit": "Edit time series",
+                    "nonNumericWarning": "Note: Non-numeric properties will be cast to numeric at query time, and may result in some values being omitted",
```

Comment:
shouldn't this be specific to strings since we only allow numeric and string? "Note: string properties will be cast to numeric at query...."

Reply:
I appreciate that it would flex to other types if we add them, but I can't imagine we want to support booleans and enums here so being specific seems pretty safe?